tools/deprecate_version/deprecate_version.py

```diff
@@ -31,6 +33,11 @@
 import github
 from git import Repo
 
+try:
+  input = raw_input  # Python 2
```

Comment:
I think the scope of this PR is more than just `print`, it seems general Python 3 compat, can you update the description?

Reply:
Fixed.